### DISCUSSION TITLE
Mutliple tmp_override directories

### DISCUSSION THREAD

**Discussion by stikeleather** [ericaltendorf/plotman#376] [Q&A] [answered]
I have 3 different tmp drives on my machine, and all are different sizes. I basically need to specify two directories for tmp_override, but plotman doesn't like when I specify more than one. Could someone provide an example of two directories in the tmp_overrides list (if possible)?

**Comment by thomann061** [comment, marked as answer]
#279 This may help you

**Reply by stikeleather** [reply]
That works! Thank you!
